README.md

```diff
@@ -0,0 +1,185 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Crypto Analysis <a name="about-project"></a>
+
+**Crypto Ananlysis** is a single page web app that fetches data from [CoinGecko API](https://www.coingecko.com/en/api/documentation). The app includes the top _100_ crypto currencies, and the details of each crypto coin. It allows user to filter the coins and navigate to details page by clicking on the selected coin.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/react-redux">Linters</a></li>
+    <li><a href="https://www.coingecko.com/en/api/documentation">CoinGeko API</a></li>
+    <li><a href="https://testing-library.com/docs/react-testing-library/intro/">React Testing Library</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **The web app shows the top 100 cryptocurrencies and their data**
+- **User can can cryptocurrents from the search box without rolling around whole page**
+- **User can view the details of the selected cryptocurrency**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://kanzatahreem.github.io/Crypto-Analysis/)
+- [Loom Video Pesentation](https://www.loom.com/share/b75aeada007d478bac3e2bf311105a51)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started with React App <a name="getting-started"></a>
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git
+- node.js
+- Wbpack
+- JavaScript
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd Crypto-Analysis
+  git clone git@github.com:KanzaTahreem/Crypto-Analysis.git
+```
+
+### Install
+
+Install this project with:
+
+```
+  cd Crypto-Analysis
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+  npm test
+```
+
+### Deployment
+
+You can deploy this project using:
+
+```
+  npm run deploy
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
```

Comment:
- [ ] OPTIONAL
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor. You can optionally also change it to _Author(s)_ to create room for future contributors.